Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:22.04
```

Comment:
We should be using ubuntu from the ECR registry ideally, it's updated more frequently